pkg/services/ngalert/api/api_alertmanager_test.go

```diff
@@ -208,6 +212,42 @@ func TestAlertmanagerConfig(t *testing.T) {
 
 		require.Equal(t, 202, response.Status())
 	})
+
+	t.Run("assert provenance status is returned", func(t *testing.T) {
+		t.Run("when routes are provisioned", func(t *testing.T) {
+			sut := createSut(t, nil)
+			rc := createRequestCtxInOrg(1)
+
+			response := sut.RouteGetAlertingConfig(rc)
+
+			require.Equal(t, 200, response.Status())
```

Comment:
Is there any data in the response that needs to be asserted here?

Reply:
Is this test even necessary with the two tests below?

Reply:
hah, nope, this can be deleted entirely. this was from a draft implementation and is totally surpassed by the tests added below. 🙃